lib/cipher/registry.ts

```
@@ -126,13 +126,13 @@ export const CIPHER_REGISTRY: CipherDefinition[] = [
-    id: "bacon",
-    name: "Bacon's Cipher",
+    id: "trithemius",
+    name: "Trithemius Cipher",
     category: "classical",
     description:
-      'A steganographic cipher encoding each letter as a 5-bit binary string using two symbols (A and B), designed to hide messages in innocent-looking text.',
+      "The simplest polyalphabetic cipher: each letter is shifted by its position index, producing an ever-increasing Caesar shift",
     defaultKey: "",
-    defaultInput: "HELLO",
+    defaultInput: "Hello, World!",
     securityStatus: "broken",
```

Biome (2.5.7)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)